### PR TITLE
I18n: Improve i18n and multi-platform display of FSRS Simulator

### PR DESCRIPTION
**Pull request by OuOu2021**
## Before:
image
(title being too small and not translatable)

8573ca143480867c574e7b131c822c3
(hard to read on my mobile phone, [Ankidroid 2.20.0](https://github.com/ankidroid/Anki-Android/releases/tag/v2.20.0))

## After (Chinese translations are not included by this pull request):
image
image

## Descriptions:
1. Make the y axis title of FSRS Simulator translatable.
2. Use `rem` font size unit and enlarge the title to make it more clear for mobile devices. 

But since I am not a Kotlin Android developer, I'm not able to deploy this to Ankidroid to check the actual effect, so I set it as a draft and hope someone can help me.

-----------
2024.12.12 update:
1. I18n for 4 more strings
2. Delete container wrapper of fsrs simulator graph
image

### CODE REVIEW

**Review by mikehardy — APPROVED** [review]
seems reasonable to me - the translations at least seem like an easy +1
altering styles/layout is kind of trial and error and the results seem fine to me as well

File: ftl/core/deck-config.ftl [hunk 1]
```diff
@@ -526,3 +526,5 @@ deck-config-invalid-weights = Parameters must be either left blank to use the de
 deck-config-fsrs-on-all-clients =
     Please ensure all of your Anki clients are Anki(Mobile) 23.10+ or AnkiDroid 2.17+. FSRS will
     not work correctly if one of your clients is older.
+deck-config-fsrs-simulator-y-axis-title-time = Review Time per day
```
Comment (dae) [inline ftl/core/deck-config.ftl:529]: The correct title case in English is "Review Time per Day", which looks a bit weird. How about the following instead?

```suggestion
deck-config-fsrs-simulator-y-axis-title-time = Review Time/Day
```

File: ftl/core/deck-config.ftl [hunk 2]
```diff
@@ -526,3 +526,5 @@ deck-config-invalid-weights = Parameters must be either left blank to use the de
...
+deck-config-fsrs-simulator-y-axis-title-count = Review Count per day
```
Comment (dae) [inline ftl/core/deck-config.ftl:530]:
```suggestion
deck-config-fsrs-simulator-y-axis-title-count = Review Count/Day
```

**Review by dae — COMMENTED** [review]
Thank you! Happy to merge once the conflict is fixed.

**Comment by dae** [comment]
To check on Android, you can use

```
$ ANKI_API_HOST=0.0.0.0 ./run
```

Then surf to http://<computer ip>:40000/deck-options/1 on your phone

**Comment by brishtibheja** [comment]
In some cases I get the y-axis title overrun with the text. I think it'd be better to have the borders around the simulator graph removed. It frees up some space and it'd look less cramped too. Can you do it in this PR?

Screenshot_2024-12-12-00-13-29-90_a9eef3a2a561b80d5c76daebd0f9a14c

**Comment by brishtibheja** [comment]
Unrelated: as you're working on stats recently, you might be interested in [the idea here](https://forums.ankiweb.net/t/feature-request-estimated-total-knowledge-over-time/53036/7?u=sorata), which would be a graph for how total knowledge changes overtime in the simulator.

**Comment by dae** [comment]
This is marked as a draft - do you plan to make further changes?

File: ftl/core/deck-config.ftl [hunk 3]
```diff
@@ -461,6 +461,12 @@ deck-config-answer-hard = Answer Hard
 deck-config-answer-good = Answer Good
 deck-config-days-to-simulate = Days to simulate
 deck-config-desired-retention-below-optimal = Your desired retention is below optimal. Increasing it is recommended.
+deck-config-fsrs-simulator-y-axis-title-time = Review Time per day
```
Comment (dae) [inline ftl/core/deck-config.ftl:464]:
```suggestion
deck-config-fsrs-simulator-y-axis-title-time = Review Time/Day
```

File: ftl/core/deck-config.ftl [hunk 4]
```diff
@@ -461,6 +461,12 @@ deck-config-answer-hard = Answer Hard
...
+deck-config-fsrs-simulator-y-axis-title-count = Review Count per day
```
Comment (dae) [inline ftl/core/deck-config.ftl:465]:
```suggestion
deck-config-fsrs-simulator-y-axis-title-count = Review Count/Day
```

**Comment by dae** [comment]
Thank you for your work on this!

**Comment by OuOu2021** [comment]
> Thank you for your work on this!

[Pontoon](https://i18n.ankiweb.net/projects/core/) is not updated, is there something I am missing?

**Comment by dae** [comment]
I've synced translations since then, so please let me know if it's still not updated.

**Pull request merged**